packages/x-date-pickers/src/DateField/tests/selection.DateField.test.tsx

```diff
@@ -77,14 +77,16 @@ describe('<DateField /> - Selection', () => {
       act(() => {
         input.focus();
       });
-      clock.runToLast();
       input.select();
 
-      expectFieldValueV6(input, 'MM/DD/YYYY');
-      expect(getCleanedSelectedContent()).to.equal('MM/DD/YYYY');
+      await waitFor(() => {
+        expectFieldValueV6(input, 'MM/DD/YYYY');
+      });
+
+      expect(getCleanedSelectedContent()).to.equal('MM');
```

Comment:
Previous check seemed wrong. If I do it on the browser the selection is `MM`

Reply:
Isn't the missing/removed `input.select()` to blame for here? 🤔

Reply:
Uh yeah, it seems like it, thanks